### PR TITLE
Fix yaml snippets for saucectl config

### PR DESCRIPTION
**Pull request by mhan83**
### Description

This (partially) reverts commit faf8e64b00d0999c6777ce0cebc59d4c27bbac61.

yaml code snippets had their whitespace stripped in the above commit making most of the invalid.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/web-apps/automated-testing/_partials/_advanced-cypress.md [hunk 1]
```diff
@@ -3,12 +3,10 @@
...
-it('.type() - type into a DOM element', { tags: ['smoke'] }, () => {
-cy
-.get('.action-email')
-.type('fake@email.com')
-.should('have.value', 'fake@email.com')
-})
+	it('.type() - type into a DOM element', { tags: ['smoke'] }, () => {
+    cy.get('.action-email')
+        .type('fake@email.com').should('have.value', 'fake@email.com')
+  })
```
Comment (FriggaHel) [inline docs/web-apps/automated-testing/_partials/_advanced-cypress.md:9]: I guess that should be more this way or something like this or similar.
Note: In current change, 1st blank (before `it` is a `<tab>`.

```suggestion
    it('.type() - type into a DOM element', { tags: ['smoke'] }, () => {
      cy.get('.action-email')
        .type('fake@email.com').should('have.value', 'fake@email.com')
    })
```

File: docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md [hunk 2]
```diff
@@ -289,9 +287,9 @@ launchOrder: fail rate
 A property containing one or more environment variables that are global for all tests suites in this configuration. Values set in this global property will overwrite values set for the same environment variables set at the suite level.
 
 ```yaml
-env:
-hello: world
-my_var: $MY_VAR # You can also pass through existing environment variables through parameter expansion
+  env:
+    hello: world
+    my_var: $MY_VAR  # You can also pass through existing environment variables through parameter expansion
```
Comment (FriggaHel) [inline docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md:292]: Useless extra space
```suggestion
    my_var: $MY_VAR # You can also pass through existing environment variables through parameter expansion
```

File: docs/web-apps/automated-testing/cypress/yaml/v1alpha.md [hunk 3]
```diff
@@ -898,14 +898,14 @@ mode: 'sauce'
 Provides details related to the Cypress test configuration that are relevant for this test suite.
 
 ```yaml {5}
```
Comment (FriggaHel) [inline docs/web-apps/automated-testing/cypress/yaml/v1alpha.md:900]: Isn't the `{5}` bringing any side-effects ?

**Review by FriggaHel — APPROVED** [review]

**Review by tianfeng92 — APPROVED** [review]

File: docs/web-apps/automated-testing/cypress/yaml/v1alpha.md (hunk 3 above)
Comment (mhan83) [reply docs/web-apps/automated-testing/cypress/yaml/v1alpha.md:900]: Huh, the `{5}` is to highlight a line in the snippet. 🤯

Comment (mhan83) [reply docs/web-apps/automated-testing/cypress/yaml/v1alpha.md:900]: https://docs.saucelabs.com/web-apps/automated-testing/cypress/yaml/v1alpha/#config

**Review by aliciakrebs — APPROVED** [review]

**Pull request merged**